.github/workflows/build-push-image-stage.yml

```diff
@@ -0,0 +1,36 @@
+name: Tutor Build Action
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.stage'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Get current time
+        uses: josStorer/get-current-time@v2
+        id: current-time
+        with:
+          format: YYYY-MM-DD-HH-mm
+
+      - name: Build and push Image
+        uses: Pearson-Advance/tutor-build-image-action@0.1.1
```

Comment:
Same comment: The lastest tag of tutor tutor-build-image-action is 0.2.0, it's that right?

Reply:
Exactly right!
I'm going to change the tag. Thanks @anfbermudezme